src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -43,40 +39,38 @@ class SettingsLayout extends React.Component {
               </Box>
               <SettingsSearch routes={routes} router={router} params={params} />
             </Flex>
-          </Container>
-        </SettingsHeader>
+          </SettingsHeader>
 
-        {/* this is div wrapper is required, else content won't stretch horizontally */}
-        <ContentContainerWrapper>
-          <Container>
+          <MaxWidthContainer>
             {typeof renderNavigation === 'function' && (
               <SidebarWrapper>{renderNavigation()}</SidebarWrapper>
             )}
             <Content>{children}</Content>
-          </Container>
-        </ContentContainerWrapper>
-        <Footer />
+          </MaxWidthContainer>
+          <Footer />
+        </SettingsColumn>
       </React.Fragment>
     );
   }
 }
-export default SettingsLayout;
 
-const Container = styled(Flex)`
+const MaxWidthContainer = styled(Flex)`
   max-width: ${p => p.theme.settings.containerWidth};
-  margin: 0 auto;
-  padding: 0 ${p => p.theme.grid * 2}px;
-`;
-
-// this wrapper is required, else content won't stretch horizontally
-const ContentContainerWrapper = styled(Box)`
-  flex: 1; /* so this stretches vertically so that footer is fixed at bottom */
 `;
 
 const SidebarWrapper = styled(Box)`
   flex-shrink: 0;
   width: ${p => p.theme.settings.sidebarWidth};
-  margin-top: 7px;
+  background: #fff;
+  border-right: 1px solid ${p => p.theme.borderLight};
+  padding: 30px;
+`;
+
+const SettingsColumn = styled('div')`
```

Comment:
This should be flex with direction column and `MaxWidthContainer` should have `flex: 1`, that'll make footer stick to bottom

Reply:
Ty!